gatsby/content/blog/2022/11/2022-11-22-synapse-1.72.mdx

```diff
@@ -0,0 +1,128 @@
+---
+date: '2022-11-22'
+title: Synapse 1.72 released
+categories:
+- Releases
+author: Mathieu Velten
+---
+
+It's not Christmas yet, but it's time for a new release! Say hello to [Synapse
+1.72](https://github.com/matrix-org/synapse/releases/tag/v1.72.0).
+
+It seems like this blog didn't self update through AI for the 2 preceding
+updates so this post will cover 1.70 to 1.72, sorry if this is a bit long but
+it's been a while :)
+
+## Announcements
+
+### Dropping support for PostgreSQL 10
+
+PostgreSQL 10 has reached End Of Life: Synapse will no longer support it
+beginning with version 1.72 so please upgrade your database if you have not
+already done so.
+
+### Legacy Prometheus metric names disabled and to be removed
+
+In previous versions of Synapse, some Prometheus metrics were emitted under two
+different names, an older name that was non-compliant with OpenMetrics and
+Prometheus conventions and a new compliant name.
+
+Synapse 1.71 and later have the old metric names switched off by default.
+
+For now it's still possible to get them by using `enable_legacy_metrics: true`,
+however server administrators still using legacy metric names are highly
+encouraged to migrate, as they will be removed in 1.73.
+
+You can find the full list of renamed metrics
+[here](https://matrix-org.github.io/synapse/v1.72/metrics-howto.html#renaming-of-metrics--deprecation-of-old-names-in-12).
+
+### Changes to the events received by application services (interest)
+
+Following the implementation of
+[MSC3905](https://github.com/matrix-org/matrix-spec-proposals/pull/3905),
+Synapse now only considers local users to be interesting to application
+services. In other words, the users namespace regex in an app service's
+registration file is only applied against local users of the homeserver.
+
+Please note, this probably doesn't affect the expected behavior of your
+application service, since an interesting local user in a room still means all
+messages in the room (from local or remote users) will still be considered
+interesting.
+
+You can find a bit more info in the MSC and in the (upgrade
```

Comment:
This markdown link seems to be the wrong way around. :)

Reply:
Bleh, so it is, /me fixes

Thanks for pointing it out!

Reply:
Fixed in https://github.com/matrix-org/matrix.org/commit/5ec81c25bd7296c659e723a01c89e0796e5cfc2e